runid:qtcoojditc
************* Module solver.__main__
solver/__main__.py:41:0: E0611: No name 'DigitSolver' in module 'solver' (no-name-in-module)
************* Module solver.util.__init__
solver/util/__init__.py:4:0: C0301: Line too long (153/100) (line-too-long)
solver/util/__init__.py:6:0: C0301: Line too long (264/100) (line-too-long)
solver/util/__init__.py:8:0: C0301: Line too long (197/100) (line-too-long)
solver/util/__init__.py:14:0: C0301: Line too long (154/100) (line-too-long)
solver/util/__init__.py:19:0: C0305: Trailing newlines (trailing-newlines)
************* Module solver.util.pretty_solve.__init__
solver/util/pretty_solve/__init__.py:1:0: C0305: Trailing newlines (trailing-newlines)
************* Module solver.util.pretty_solve.__main__
solver/util/pretty_solve/__main__.py:1:0: C0114: Missing module docstring (missing-module-docstring)
solver/util/pretty_solve/__main__.py:14:0: C0413: Import "from solver.solver import DigitSolver" should be placed at the top of the module (wrong-import-position)
solver/util/pretty_solve/__main__.py:19:0: C0116: Missing function or method docstring (missing-function-docstring)
solver/util/pretty_solve/__main__.py:23:4: C0103: Function name "mf" doesn't conform to snake_case naming style (invalid-name)
solver/util/pretty_solve/__main__.py:32:12: R1722: Consider using 'sys.exit' instead (consider-using-sys-exit)
solver/util/pretty_solve/__main__.py:72:4: E1101: Instance of 'DigitSolver' has no 'printer_setter' member (no-member)
solver/util/pretty_solve/__main__.py:87:38: W0108: Lambda may not be necessary (unnecessary-lambda)
solver/util/pretty_solve/__main__.py:19:0: R0915: Too many statements (51/50) (too-many-statements)
************* Module solver.util.solve_auto.solve_auto
solver/util/solve_auto/solve_auto.py:18:0: E0401: Unable to import 'selenium.webdriver.common.by' (import-error)
solver/util/solve_auto/solve_auto.py:19:0: E0401: Unable to import 'selenium.webdriver.support.wait' (import-error)
solver/util/solve_auto/solve_auto.py:20:0: E0401: Unable to import 'selenium.webdriver.support' (import-error)
solver/util/solve_auto/solve_auto.py:21:0: E0401: Unable to import 'selenium.webdriver.remote.webdriver' (import-error)
solver/util/solve_auto/solve_auto.py:60:4: W0702: No exception type(s) specified (bare-except)
solver/util/solve_auto/solve_auto.py:107:4: W0702: No exception type(s) specified (bare-except)
solver/util/solve_auto/solve_auto.py:124:4: W0702: No exception type(s) specified (bare-except)
************* Module solver.util.solve_auto.__init__
solver/util/solve_auto/__init__.py:1:0: C0305: Trailing newlines (trailing-newlines)
************* Module solver.util.solve_auto.__main__
solver/util/solve_auto/__main__.py:51:0: C0301: Line too long (152/100) (line-too-long)
solver/util/solve_auto/__main__.py:48:0: E0401: Unable to import 'selenium' (import-error)
solver/util/solve_auto/__main__.py:49:0: E0401: Unable to import 'selenium.webdriver.chrome.options' (import-error)
solver/util/solve_auto/__main__.py:51:0: E0401: Unable to import 'solve_auto' (import-error)
solver/util/solve_auto/__main__.py:62:0: C0413: Import "from solver.solver import DigitSolver" should be placed at the top of the module (wrong-import-position)
solver/util/solve_auto/__main__.py:147:0: R0914: Too many local variables (17/15) (too-many-locals)
solver/util/solve_auto/__main__.py:163:55: C0103: Variable name "f" doesn't conform to snake_case naming style (invalid-name)
solver/util/solve_auto/__main__.py:189:12: E1101: Instance of 'DigitSolver' has no 'printer_setter' member (no-member)
solver/util/solve_auto/__main__.py:189:34: W0108: Lambda may not be necessary (unnecessary-lambda)
solver/util/solve_auto/__main__.py:189:44: W0640: Cell variable solution defined in loop (cell-var-from-loop)

-----------------------------------
Your code has been rated at 7.96/10

